bpf/lib/nodeport.h

```diff
@@ -858,6 +858,9 @@ static __always_inline int nodeport_lb6(struct __ctx_buff *ctx,
 
 #if defined(ENABLE_L7_LB)
 		if (lb6_svc_is_l7loadbalancer(svc) && svc->l7_lb_proxy_port > 0) {
+#if __ctx_is == __ctx_xdp
+			return CTX_ACT_OK;
+#endif
```

Comment:
small nit: could we move __ctx_is == __ctx_xdp into a small helper?
```
if (layer_is_xdp())
       return CTX_ACT_OK;
```

Reply:
Good idea! I will do it in a follow-up, as we might want to replace `__ctx_is == __ctx_skb` too.

Reply:
ack, would be nice to have some consistent way for the two.

Reply:
https://github.com/cilium/cilium/issues/23008